test/t8_test_cmesh_copy.c

```diff
@@ -37,40 +47,249 @@ test_cmesh_committed (t8_cmesh_t cmesh)
   SC_CHECK_ABORT (retval == 1, "Cmesh face consistency failed.");
 }
 
+
+/*
+ *The functions t8_get_*_cmesh_testcases return the number of 
+ *testcases for a given cmesh type.
+ */
+static  int  t8_get_comm_only_cmesh_testcases():
+{
+  /*number of functions that only take comm as input * number of comm */
+  return 10*num_comm;
+}
+
+static  int  t8_get_new_hypercube_cmesh_testcases():
+{
+  /* number of element types * number of comm*3 binary variables(do_bcast, do_partition,periodic) */
+  return T8_ECLASS_COUNT*num_comm*binary*binary*binary;
+}
+
+static  int  t8_get_new_empty_cmesh_testcases():
+{
+  /* number of comm* 1 binary variable(do_partition)* possible dimensions(check dim 0 to 4, therefore we need to add 2 to dim)*/
+  return num_comm*binary*(dim+2);
+}
+
+static  int  t8_get_new_from_class_cmesh_testcases():
+{
+  /* number of element types * number of comm */
+  return T8_ECLASS_COUNT*num_comm;
+}
+
+static  int  t8_get_new_hypercube_hybrid_cmesh_testcases():
+{
+  /* possible dim * number of comm*2 binary variables(do_partition,periodic) */
+  return 4*num_comm*binary*binary;
+}
+
+static  int t8_get_new_periodic_cmesh_testcases():
+{
+  /*  number of comm * possible dim */
+  return num_comm*dim;
+}
+
+static  int  t8_get_new_bigmesh_cmesh_testcases():
+{
+  /*  number of element types * number of trees * number of comm */
+  return T8_ECLASS_COUNT*max_num_of_trees*num_comm;
+}
+
+static  int  t8_get_new_prism_cake_cmesh_testcases():
+{
+  /*  number of comm * number of prisms */
+  return num_comm*max_num_of_prisms;
+}
+
+static  int t8_get_new_disjoint_bricks_cmesh_testcases():
+{
+  /*  TO DO: add comment*/
+  return 20*20*20*binary*binary*binary*num_comm;
```

Comment:
What does the 20 stand for?
I guess number of cases per dimension?
Should be a macro.